generate-protos.sh

```diff
@@ -2,8 +2,8 @@
 
 set -ex
 PROTOC_VERSION="$(protoc --version)"
-if [ "$PROTOC_VERSION" != "libprotoc 3.6.0" ]; then
-	echo "Must have libprotoc version 3.6.0."
+if [ "$PROTOC_VERSION" != "libprotoc 3.6.0" || "$PROTOC_VERSION" != "libprotoc 3.6.1" ]; then
```

Comment:
do we need this `||` condition or should we just check for `3.6.1`?
also do we need to pin `protoc` to `3.6.1` in `dev-requirements.txt` file?

Reply:
If we remove the `||`, then we would require contributors to update from `3.6.0` to 3.6.1`. I'd prefer not to ask people to update to `3.6.1` because it's not necessary.

The python package does not have `protoc` and I had to install it separately using `brew install protobuf@3.6`. Secondly, in `setup.py`, we already specify `protobuf>=3.6.0` which means that new contributors will be running `3.11.1`.  Therefore, I don't think we should add it to `dev-requireents.txt`

Furthermore, looking at the protobuf [releases versions(https://github.com/protocolbuffers/protobuf/tags?after=v3.9.0), we only need to handle 3.6.1 and 3.6.0 right now. The latest protobuf version is 3.11.x and it does not past the test script.

Reply:
I see. Thanks for clarifying. In that case, can rewrite the error message to something like--

```
Required libprotoc versions to be 3.6.0 or 3.6.1 (preferred).
```

Or something like that?

Reply:
Of course, that makes a lot of sense.